docs/default_ignore_unused_list.md

```diff
@@ -0,0 +1,51 @@
+## Default Ignore Unused List
+
+### Formatting Tools
+
+- autopep8
+- black
+- codespell
+- isort
+- pyformat
+- yapf
+
+### Linting Tools
+
+- flake8
+- mccabe
+- mypy
+- pyflakes
+- pylint
+- pyright
+- ruff
+
+### Security Tools
+
+- bandit
+
+### Documentation Tools
+
+- myst-parser
+- recommonmark
+- sphinx
+- sphinx-rtd-theme
+
+### Testing Tools
+
+- coverage
+- fawltydeps
+- nox
+- pre-commit
+- pytest
+- tox
+
+### Building and Packaging Tools
+
+- twine
+- wheel
+
+### Utility Tools
+
+- pydocstyle
+- rope
+- unify
```

Comment:
I don't like duplicating this list between code and documentation. One will invariably fall out of sync with the other. Can we link directly to the `DEFAULT_IGNORE_UNUSED` in `fawltydeps/settings.py` instead?